src/content/docs/develop/configuration-files.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Configuration Files
+sidebar:
+  order: 1
+---
+
+Since Tauri is a toolkit for building applications there can be many files to configure project settings. Some common files that you may run across are `tauri.conf.json`, `package.json` and `Cargo.toml`. We briefly explain each on this page to help point you in the right direction for which file to modify.
```

Comment:
```suggestion
Since Tauri is a toolkit for building applications there can be many files to configure project settings. Some common files that you may run across are `tauri.conf.json`, `package.json` and `Cargo.toml`. We briefly explain each on this page to help point you in the right direction for which files to modify.
```